engine/classes/ElggPlugin.php

```diff
@@ -1151,7 +1147,7 @@ protected function deactivateEntities() {
 		
 		foreach ($spec as $entity) {
 			if (isset($entity['type'], $entity['subtype'], $entity['class'])) {
-				update_subtype($entity['type'], $entity['subtype']);
+				elgg_set_entity_class($entity['type'], $entity['subtype']);
```

Comment:
is this still necessary? there is no more persistence right? Or do we want it unset during runtime...

Reply:
Better to leave it in in case some logic between deactivate and page reload

Reply:
fine by me